packages/docusaurus-theme-classic/src/theme/Toggle/index.js

```diff
@@ -10,11 +10,10 @@ import Toggle from 'react-toggle';
 
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 
-import classnames from 'classnames';
-import styles from './styles.module.css';
+import './styles.css';
 
-const Moon = () => <span className={classnames(styles.toggle, styles.moon)} />;
-const Sun = () => <span className={classnames(styles.toggle, styles.sun)} />;
+const Moon = () => <span className="toggle moon" />;
```

Comment:
Keep these as CSS modules. `toggle` is an *extremely* common classname and you won't want to risk collisions.